### PR TITLE
[WIP] Update dependencies

### PR DESCRIPTION
**Pull request by terorie**
Ran `npm upgrade`, `npm audit fix` and pushed `multiaddr` to `6.0.0`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,81 +27,81 @@
...
+    "ipfs-unixfs": "^0.1.16",
+    "ipld": "^0.20.1",
     "ipld-explorer-components": "^1.0.0",
     "is-binary": "^0.1.0",
-    "is-ipfs": "^0.4.2",
+    "is-ipfs": "^0.4.8",
     "milliseconds": "^1.0.3",
-    "money-clip": "^2.1.0",
-    "multiaddr": "^5.0.2",
+    "money-clip": "^3.0.0",
```
Comment (olizilla) [inline package.json]: Note to self: This is a major, but from the changelog it looks like it was a packaging issue https://github.com/HenrikJoreteg/money-clip#change-log ... is fine.

File: package.json [hunk 2]
```diff
@@ -27,81 +27,81 @@
...
+    "react": "^16.6.3",
+    "react-ace": "^6.2.0",
     "react-chartjs-2": "^2.7.4",
     "react-copy-to-clipboard": "^5.0.1",
     "react-country-flag": "^1.0.1",
-    "react-debounce-render": "^4.0.3",
-    "react-dnd": "^5.0.0",
-    "react-dnd-html5-backend": "^5.0.1",
-    "react-dom": "^16.4.2",
+    "react-debounce-render": "^5.0.0",
```
Comment (olizilla) [inline package.json]: Yes please: https://github.com/podefr/react-debounce-render#500---oct-04-2018

File: package.json [hunk 3]
```diff
@@ -27,81 +27,81 @@
...
+    "react-dnd": "^7.0.2",
```
Comment (olizilla) [inline package.json]: https://github.com/react-dnd/react-dnd/releases
Need to check the drag and drop to add files on the files page, we lack a test for that

File: package.json [hunk 4]
```diff
@@ -27,81 +27,81 @@
...
-    "@storybook/addons": "^3.4.10",
-    "@storybook/react": "^3.4.10",
-    "@svgr/cli": "^2.3.0",
+    "@storybook/addon-a11y": "^3.4.11",
+    "@storybook/addon-actions": "^3.4.11",
+    "@storybook/addon-knobs": "^3.4.11",
+    "@storybook/addon-links": "^3.4.11",
+    "@storybook/addons": "^3.4.11",
+    "@storybook/react": "^3.4.11",
+    "@svgr/cli": "^4.1.0",
```
Comment (olizilla) [inline package.json]: https://github.com/smooth-code/svgr/blob/master/packages/cli/CHANGELOG.md
Note to self: Need to check our generated Icon components, created via `npm run build:icons`

Comment (terorie) [reply package.json]: Output of `build:icons` committed and pushed

File: package.json (hunk 1 above)
Comment (terorie) [reply package.json]: Yeah, it only has [minimal changes](https://github.com/HenrikJoreteg/money-clip/commit/0118bbabcd6f4108ab279cb18fb45b35d701b3c2) that don't affect our use (`import { getConfiguredCache } from 'money-clip'` in `bundle-cache.js`)

**Comment by hacdias** [comment]
Thanks for the effort @terorie! I'm now updating all the old deps.

**Pull request closed without merging**